### PR TITLE
Debenick,Tchadde | Change  link to a dropdown

### PR DESCRIPTION
**Pull request by debenick17**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1892,7 +1906,20 @@
       "integrity": "sha512-JVwXMr9nHYTUXsBFKUqhJwvlcYU/blreOEUkhNR2eXZIvwd+c+o5V4MgDPKWnMS/56awN3TRzIP+KoPn+roQtg==",
       "dev": true,
       "requires": {
+<<<<<<< Updated upstream
```
Comment (jlstrater) [inline package-lock.json:1909]: 😞 It looks like you have some unresolved merge conflicts in here. Let me know if you need some help resolving them.

File: src/app/internship/internship.component.html [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+</div>
+<div class="section section-we-do-presentation">
+  <div class="container">
+      <div class="container">
+          <div class="title-area">
+              <p class="description" style="text-align:justify;">The groovy Summer Internship organised by GR8DI, is a program initiated to help students in the English speaking part of Cameroon to complete their degree program.Due to the increased violence in Cameroon, several of our students have been left without summer internships. To help solve this issue, GR8DI with the help of Jennifer Strater and Kamwa Fosso Arcel decided to organise Summer Internships for students to learn and build the GR8DI website. The first badge of this program mainly consisted of students of the College Of Technology (University Of Buea). These students will be mentored by Kamwa Fosso Arcel Raulain (aka kraulain).</p>
+          </div>
+          <div class="row Welcome text-center">
+              <div class="col-12">
+                <h2 class="display-4">Meet the first badge of GR8DI Internship Program</h2>
```
Comment (jlstrater) [inline src/app/internship/internship.component.html:21]: badge -> batch

File: src/app/internship/internship.component.html [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+                <div class="separator">♦</div>
+            </div>
+        </div>
+    </div>
+</div>
+<div class="section section-we-do-presentation">
+  <div class="container">
+      <div class="container">
+          <div class="title-area">
+              <p class="description" style="text-align:justify;">The groovy Summer Internship organised by GR8DI, is a program initiated to help students in the English speaking part of Cameroon to complete their degree program.Due to the increased violence in Cameroon, several of our students have been left without summer internships. To help solve this issue, GR8DI with the help of Jennifer Strater and Kamwa Fosso Arcel decided to organise Summer Internships for students to learn and build the GR8DI website. The first badge of this program mainly consisted of students of the College Of Technology (University Of Buea). These students will be mentored by Kamwa Fosso Arcel Raulain (aka kraulain).</p>
```
Comment (jlstrater) [inline src/app/internship/internship.component.html:17]: Add a space before Due.

**Review by jlstrater — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 1 above)
Comment (debenick17) [reply package-lock.json:1909]: Yeah, I need help resolving them

File: src/app/internship/internship.component.html (hunk 2 above)
Comment (debenick17) [reply src/app/internship/internship.component.html:21]: Oh thanks

File: src/app/internship/internship.component.html (hunk 3 above)
Comment (debenick17) [reply src/app/internship/internship.component.html:17]: Alright

**Pull request merged**